src/Microsoft.Azure.WebJobs.Host/Scale/ScaleManager.cs

```diff
@@ -224,8 +223,8 @@ public async Task<ScaleStatus> GetScaleStatusAsync(ScaleStatusContext context)
                     if (!_targetScalersInError.Contains(scalerUniqueId))
                     {
                         string assemblyName = GetAssemblyName(scaler.GetType());
-                        string flag = _hostingConfig.Value.GetFeature(assemblyName);
-                        if (flag == "1")
+                        bool featureEnabled = await _featureManager.IsEnabledAsync(assemblyName);
```

Comment:
We should find a way to fix the feature names here, so for the code checked into the SDK meaningful feature names are used, rather than assembly name. Perhaps the scaler host can do that modification when it adds the feature flags to config?

Reply:
TBS is enabling right now, SC also relies on the same flag name(assembly name).

Changing the flag name will cause the TBS delay (we did commit ETA as mid Feb):
1. SC needs to be updated to use the new flag name.
2. Stamps configs needs to be updated - some stamps already have TBS.
3. Function host with new web jobs sdk reference needs to be releases for TBS+RTS.